src/test/java/bio/terra/common/CloudPlatformWrapperTest.java

```diff
@@ -0,0 +1,29 @@
+package bio.terra.common;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import bio.terra.common.category.Unit;
+import bio.terra.model.CloudPlatform;
+import java.util.Map;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag(Unit.TAG)
+class CloudPlatformWrapperTest {
+
+  @Test
+  void choose_Gcp() {
+    assertEquals(
+        "GCP",
+        CloudPlatformWrapper.of(CloudPlatform.GCP)
+            .choose(Map.of(CloudPlatform.GCP, () -> "GCP", CloudPlatform.AZURE, () -> "Azure")));
```

Comment:
I feel like the `Map` form is still a bit too much. Do you think it would be much worse to make it entirely positional? We would still have our unit tests to verify that the correct path is used, so the main downside would be readability.
```suggestion
            .choose(() -> "GCP", () -> "Azure"));
```

An intermediate approach wold be move the `Map` constructor argument list to `choose()`:
```suggestion
            .choose(CloudPlatform.GCP, () -> "GCP", CloudPlatform.AZURE, () -> "Azure"));
```